src/components/scene/keyboard-shortcuts.js

```diff
@@ -11,23 +11,27 @@ module.exports.Component = registerComponent('keyboard-shortcuts', {
     var self = this;
     var scene = this.el;
 
-    this.listener = window.addEventListener('keyup', function (event) {
+    this.listener = function (event) {
```

Comment:
Can we make a method `onKeyup` following the same pattern found in the code base? e.g: https://github.com/aframevr/aframe/blob/master/src/components/scene/inspector.js#L50